apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -406,9 +407,7 @@ export class PretixPipeline implements BasePipeline {
   }
 
   public async triggerSemaphoreGroupUpdate(): Promise<void> {
-    const data = await this.semaphoreGroupData();
     return traced(LOG_NAME, "triggerSemaphoreGroupUpdate", async (_span) => {
-      await this.semaphoreGroupProvider?.update(data);
```

Comment:
This was a result of a poorly-executed merge. The result of the function is unaffected, but the data was being polled twice unnecessarily.